### PR TITLE
Adding logout functionality.

### PR DESCRIPTION
**Pull request by kshamadeshmukh**

### CODE REVIEW

File: src/Service/Users.php [hunk 1]
```diff
@@ -132,4 +132,41 @@ public function getUserData($username, $authToken, array $options = []) {
...
+
+      $requestOptions = [
+        'headers' => [
+          'authToken' => $authToken,
+        ],
+        'body' => $postBody,
+      ];
+
+      $apiOptions = $this->config->get('logout');
+      $apiOptions['uri_template_options'] = array();
```
Comment (dakkusingh) [inline src/Service/Users.php]: Short array syntax

File: src/Service/Users.php [hunk 2]
```diff
@@ -132,4 +132,41 @@ public function getUserData($username, $authToken, array $options = []) {
...
 
+  /**
+   * Logs out user from openAM using authToken.
+   *
+   * @param string $authToken
+   *   Auth token to be used while loging out from openAM.
+   * @param array $options
+   *   Additional options for the guzzle request. e.g. proxy settings.
+   *
+   * @return bool|mixed|null|\Psr\Http\Message\StreamInterface
```
Comment (dakkusingh) [inline src/Service/Users.php]: not sure if this return makes sense?

**Review by dakkusingh — CHANGES_REQUESTED** [review]

**Comment by dakkusingh** [comment]
@passionatepoorna are you able to make minor amends to this too so we can merge this also?

File: src/Service/Users.php (hunk 1 above)
Comment (snwalkunde) [reply src/Service/Users.php]: Done

File: src/Service/Users.php (hunk 2 above)
Comment (snwalkunde) [reply src/Service/Users.php]: Updated the `@return`

**Pull request merged**